coalib/output/Logging.py

```diff
@@ -1,10 +1,26 @@
 from datetime import datetime
+from collections import defaultdict
 import json
 import io
 import logging
 import logging.config
 
 
+class CounterHandler(logging.Handler):
+    """
+    A logging handler which counts the number of calls
+    for each logging level.
+    """
+    level_called = None
+
+    def __init__(self, *args, **kwargs):
+        super(CounterHandler, self).__init__(*args, **kwargs)
+        CounterHandler.level_called = defaultdict(lambda: 0)
```

Comment:
@NiklasMM Well... I was just thinking about that `CounterHandler` could maybe be derived and used for additional logging to other interfaces like web or whatever, with explicit extra messages for those interfaces, what would cause multiple instantiation...

**CC** @sils